oauth2client/service_account.py

```diff
@@ -27,14 +27,14 @@
 from oauth2client import GOOGLE_TOKEN_URI
 from oauth2client import util
 from oauth2client._helpers import _from_bytes
-from oauth2client.client import _apply_user_agent
-from oauth2client.client import _initialize_headers
 from oauth2client.client import _UTCNOW
 from oauth2client.client import AccessTokenInfo
 from oauth2client.client import AssertionCredentials
-from oauth2client.client import clean_headers
 from oauth2client.client import EXPIRY_FORMAT
 from oauth2client.client import SERVICE_ACCOUNT
+from oauth2client.transport import _apply_user_agent
```

Comment:
I am going to rip out the equivalent of `wrap_http_for_auth` from `service_account.py` in a future PR. This will solve that problem. On the subject, `clean_headers` should have never been public in `oauth2client.client`.